### PR TITLE
[USING] Chapter on Receive 

### PR DESCRIPTION
**Pull request by MaxHillebrand**
this is a work in progress branch to write a standalone chapter on the `Receive` tab in Wasabi. So address generation, labeling and all the privacy fuckups that might happen.

There already exist quite a thorough FAQ section, I might borrow some from there and re-arrange it.

What is written is ready for review, but it's still a work in progress and everything might get re-written...

### CODE REVIEW

File: docs/using-wasabi/Receive.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+:::
+
+![](/ReceiveLabelingRequired.png)
+
+## Generating addresses step-by-step
+
+1. Start Wasabi and open the wallet that you want to receive coins into.
+2. Go to the `Receive` tab.
+3. Specify a label of the entities who know about this address.
+4. Click on `Generate Address` and it is automatically copied to your clipboard.
```
Comment (yahiheb) [inline docs/using-wasabi/Receive.md]:
```suggestion
4. Click on `Generate Address`.
```

File: docs/using-wasabi/Receive.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+If someone has only the public key, it is computationally infeasible [pretty much impossible] to calculate the private key.
+Whoever has knowledge of the private key can calculate a cryptography signature over a message, which proves that the private key was known by the signer, without actually revealing the secret.
+
+For data size efficiency, in Bitcoin the public key is [hashed](https://en.wikipedia.org/wiki/Cryptographic_hash_function) and encoded into a [bech32](/using-wasabi/BIPs.md#bip-173-base32-address-format-for-native-v0-16-witness-outputs) address.
+This address commits to the public key, which commits to the private key.
+In a Bitcoin transaction, the output creates a new unspent transaction output which is "locked up" by an address.
+The input of the transaction spends a UTXO by providing a valid signature of the public key that the address committed to.
+
+:::tip
+Wasabi wallet does not "store your money", rather it stores your private and public keys, it creates addresses for receiving bitcoin, and it signs transactions that spend your bitcoin.
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/using-wasabi/Receive.md]: yeah, that is a good point.

File: docs/using-wasabi/Receive.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+:::tip
+This is why Wasabi removes the address from the `Receive` tab as soon as it has received a coin.
+:::
+
+## The importance of labeling
+
+Satoshis are the base unit currency of the Bitcoin network and are truly fungible, meaning that any 100 000 000 satoshis are always equal to 1 bitcoin.
+Just like with gold atoms, it doesn't matter which specific gold atoms you have, as long as it's gold, it's gold.
+However, an unspent transaction output, a bitcoin, is a chunk of money and is not fungible, because it has a different amount of satoshis in it, a different public key locking it up, and a unique transaction history.
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/using-wasabi/Receive.md]: mhhh, you are right of course, but they are still different outpoints and thus different utxos... But my explanation doesn't catch this edge case...

**Comment by MaxHillebrand** [comment]
I say this is ready for merge, what do you say?
@yahiheb @RiccardoMasutti

File: docs/using-wasabi/Receive.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+If someone has only the public key, it is computationally infeasible [pretty much impossible] to calculate the private key.
+Whoever knows the private key can calculate a cryptography signature over a message, which proves that the private key was known by the signer, without actually revealing the secret.
+
+For data size efficiency, in Bitcoin the public key is [hashed](https://en.wikipedia.org/wiki/Cryptographic_hash_function) and encoded into a [bech32](/using-wasabi/BIPs.md#bip-173-base32-address-format-for-native-v0-16-witness-outputs) address.
+This address commits to the public key, which commits to the private key.
+In a Bitcoin transaction, the output creates a new unspent transaction output which is "locked up" by an address.
+The input of the transaction spends a UTXO by providing a valid signature of the public key that the address committed to.
+
+:::tip
+Wasabi Wallet does not "store your money", rather it stores your private and public keys, it creates addresses for receiving bitcoin, and it signs transactions that spend your bitcoin.
```
Comment (yahiheb) [inline docs/using-wasabi/Receive.md]:
```suggestion
Wasabi Wallet does not "store your money", rather it stores your public keys and the encrypted secret that requires a password to decrypt the private keys, it creates addresses for receiving bitcoin, and it signs transactions that spend your bitcoin.
```
@lontivero is this accurate?

File: docs/using-wasabi/Receive.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/Receive.md]: If it doesn't catch this case then it is not correct I think.
Maybe UTXOs are not fungible for other reasons but not for the ones mentioned in this sentence.

File: docs/using-wasabi/Receive.md (hunk 4 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/Receive.md]: yes, it is os

File: docs/using-wasabi/Receive.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/using-wasabi/Receive.md]: I added the index number. 
I think it is ok now.

[the remark this answers is not in the document]
Comment (yahiheb) [reply docs/using-wasabi/Receive.md]: I think now it is much better, unless someone finds another edge case lol

**Pull request merged**